platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/PlatformState.java

```diff
@@ -526,7 +551,16 @@ public long getLowestJudgeGenerationBeforeBirthRoundMode() {
      * @param lowestJudgeGenerationBeforeBirthRoundMode the lowest judge generation before the birth round mode was
      *                                                  enabled
      */
+    @Override
     public void setLowestJudgeGenerationBeforeBirthRoundMode(final long lowestJudgeGenerationBeforeBirthRoundMode) {
         this.lowestJudgeGenerationBeforeBirthRoundMode = lowestJudgeGenerationBeforeBirthRoundMode;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void bulkUpdate(Consumer<PlatformStateAccessor> updater) {
+        updater.accept(this);
```

Comment:
param missing `final` and nullity annotation

Reply:
Fixed